docs/before/accounts.md

```diff
@@ -83,36 +70,6 @@ ${SIT_PSDM_DATA}/<instrument>/<experiment>
     For more information about the S3DF, check out the PCDS documentation [here][4] and the official S3DF docs [here][5].
 
 
-### Past experiments (pre 2023)
```

Comment:
Agree with removing this, as the old cluster is mostly irrelevant for users and data analysis, but perhaps we should still have a small section to describe how to access old data? E.g. who to contact, process of moving data back from tape, etc..?

Reply:
Great point, I added a note.